projects/standalone-app/src/app/test.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { inject, Pipe, PipeTransform } from '@angular/core';
+import { Store } from '@ngrx/store';
+
+@Pipe({ name: 'test', standalone: true })
+export class TestPipe implements PipeTransform {
+  store = inject(Store);
+  transform(s: number) {
+    this.store.select('count');
+    return s * 2;
+  }
+}
```

Comment:
it would be good to add a test to `store/integration_signals.spec.ts` that validates this fix